BuildAndTest.proj

```diff
@@ -70,6 +70,7 @@
       <TestAssemblies Include="Binaries\Debug\Roslyn.Compilers.VisualBasic.Syntax.UnitTests.dll" />
       <TestAssemblies Include="Binaries\Debug\Roslyn.Compilers.VisualBasic.CommandLine.UnitTests.dll" />
       <TestAssemblies Include="Binaries\Debug\Roslyn.Diagnostics.Analyzers.FxCop.UnitTests.dll" />
+      <TestAssemblies Include="Binaries\Debug\System.Runtime.Analyzers.UnitTests.dll" />
```

Comment:
I've added the projects to Roslyn2013.sln since they don't depend on anything in 2015.